BUILD

```diff
@@ -986,9 +986,11 @@ cc_library(
     ],
 )
 
+reject_blacklisted_files = not hasattr(proto_common, "proto_lang_toolchain_rejects_files_do_not_use_or_we_will_break_you_without_mercy"),
```

Comment:
@lberki @cushon @hlopko I'd love to get your lgtm on this "feature flag" so we don't have to synchronize flipping a future `--incompatible_blacklisted_protos_requires_proto_info` (or whatever that flag will be called) with protobuf releases. See https://github.com/bazelbuild/bazel/pull/10493 and https://github.com/bazelbuild/bazel/issues/10484 for context.

Reply:
Do I understand correctly that the sole reason for this change is http://github.com/bazelbuild/bazel/issues/10484 and thus if that is fixed, this change and the flag becomes unnecessary?